README.md

```diff
@@ -47,13 +47,32 @@ Alternatively, you can configure Vault connection using next environmnt variable
 ## Preparing secrets in Vault
 
 You need to store your secrets with special naming rules.
-First of all, all secrets should use KV2 storage and have prefix `data/{app_alias}/`.
-For example, if your app has alias `sampleapp` and you want to have configuration option `ConnectionString` your secret path would be `data/sampleapp/ConnectionString`.
+First of all, all secrets should use KV2 storage and have prefix `{app_alias}/{env}`.
```

Comment:
Have you checked this with real Vault data? I remember there was a problem with reading and writing with VaultSharp secrets that are not prefixed with `data`. I'm totally ok with removing this requirement, but I'd like to be confident we don't break things.

Reply:
Yes, I checked on real data. Works. I recommend you try it too.